### PR TITLE
2-define-general-project-structure-some-kind-of-golang-monorepo

### PR DESCRIPTION
**Pull request by dominikvrbic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+
+### Building Applications
+
+```bash
+#build worker
+cd worker
+go build -o worker
+
+#build scheduler
+cd mts
```
Comment (frle10) [inline README.md:20]: Why did we call this mts?

File: README.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+
+- Ensure you have Go version 1.18 or later installed.
+- Clone the repository and navigate into the project's root directory.
+
+### Building Applications
+
+```bash
+#build worker
+cd worker
+go build -o worker
```
Comment (frle10) [inline README.md:17]: I just want to add an idea. What do you think about creating a Makefile in the root of the project that will define two custom commands, for example they could be "make run worker" and "make run mts"?

This way the commands are easier to write and both can be done from the root.

Comment (dominikvrbic) [reply README.md:17]: i would that as well but not in the scope of this ticket, make a new one for it

File: README.md (hunk 1 above)
Comment (dominikvrbic) [reply README.md:20]: cos I am an idiot, wanted to put dts but put mts but maybe we want to called it scheduler?

Comment (frle10) [reply README.md:20]: yes, that seems fine, let's do that then

Comment (dominikvrbic) [reply README.md:20]: Let's also then use that naming in the diagram as well, let's add a glossary page to the wiki to keep these thing clear as well so we can have a ubiquitous lang

Comment (frle10) [reply README.md:20]: Alright, I'll make an issue for the glossary page

**Review by frle10 — APPROVED** [review]
Looks good now

**Pull request merged**